daemon/job/active.go

```diff
@@ -496,6 +506,19 @@ func (j *ActiveSide) do(ctx context.Context) {
 		replicationReport := j.tasks.replicationReport()
 		j.promReplicationErrors.Set(float64(replicationReport.GetFailedFilesystemsCountInLatestAttempt()))
 
+		for _, attempt := range replicationReport.Attempts {
```

Comment:
I think we only need the most recent attempt here, right?
Like so:

https://github.com/zrepl/zrepl/blob/83fdffbceff9d1c8b1690dc104f9cdbb7b2f8b87/replication/report/replication_report.go#L169-L176